src/Provider/TwigServiceProvider.php

```diff
@@ -23,6 +23,14 @@ public function register(Application $app)
             $app->register(new \Silex\Provider\TwigServiceProvider());
         }
 
+        // Set authentication providers before security extension is invoked.
+        $factory = $app->raw('twig');
+        $app['twig'] = $app->share(function ($app) use ($factory) {
+            $app['security.firewall'];
```

Comment:
Eh Pimple exception made it pretty obvious...how to fix it that's another story.